python/ql/src/semmle/python/objects/ObjectAPI.qll

```diff
@@ -417,6 +422,15 @@ class ClassValue extends Value {
     predicate isDescriptorType() {
         this.hasAttribute("__get__")
     }
+    
+    /** Whether this class is a legal exception class. 
+     *  What constitutes a legal exception class differs between major versions */
+    predicate isLegalExceptionType() {
+        not this.isNewStyle() or
```

Comment:
nitpick: this `or` should be on it's own line

Reply:
Resolved via autoformatting.